build.gradle

```diff
@@ -16,17 +16,21 @@ subprojects {
 
     repositories {
         jcenter()
+        maven {
```

Comment:
Can we use `google()` instead of this block? You might need to also update the gradle version for that to be available?